schema_source/aws_serverless_graphqldatasource.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from typing_extensions import Literal
+
+from schema_source.common import BaseModel, SamIntrinsicable, get_prop
+
+properties = get_prop("sam-resource-graphqldatasource")
+
+
+# TODO: add docs
+class DeltaSync(BaseModel):
+    BaseTableTTL: str
+    DeltaSyncTableName: str
+    DeltaSyncTableTTL: str
+
+
+class DynamoDB(BaseModel):
```

Comment:
Let's have consistent names, ie this one is `DynamoDBConfig` in the model. It's confusing to see `DynamoDB` class

Reply:
I was concerned with naming it `DynamoDB` in model since it was being imported into the `sam_resources` module, and it's strictly related to DataSource. I agree though, this naming is strange, let me play around with it  more and figure out something better.